backend/src/API/coffeeChatAPI.ts

```diff
@@ -24,22 +24,12 @@ export const getCoffeeChatsByUser = async (user: IdolMember): Promise<CoffeeChat
  * A member can not coffee chat the same person from previous semesters.
  */
 export const createCoffeeChat = async (coffeeChat: CoffeeChat): Promise<CoffeeChat> => {
-  const [member1, member2] = coffeeChat.members;
-
-  if (isEqual(member1, member2)) {
+  if (isEqual(coffeeChat.submitter, coffeeChat.otherMember)) {
```

Comment:
so (1) might be easier to just compare the emails and (2) checking that calling user is the submitter or admin user should theoretically be there